impeller/display_list/skia_conversions.cc

```diff
@@ -159,6 +160,14 @@ std::vector<Matrix> ToRSXForms(const SkRSXform xform[], int count) {
   return result;
 }
 
+Path PathDataFromTextBlob(const sk_sp<SkTextBlob>& blob) {
+  if (!blob) {
+    return {};
+  }
+
+  return ToPath(skia::textlayout::Paragraph::GetPath(blob.get()));
```

Comment:
Well I'll be damned. That's a lot easier than I was expecting!

Reply:
Well, I had to wait 3 months for the API to be added so IMO not that easy 😆